crates/sui-graphql-rpc/src/config.rs

```diff
@@ -132,8 +132,39 @@ pub struct BackgroundTasksConfig {
     pub watermark_update_ms: u64,
 }
 
-#[derive(Debug)]
-pub struct Version(pub &'static str);
+#[derive(Copy, Clone, Debug)]
+pub struct Version {
+    /// The major version is set as the year
+    pub major: &'static str,
+    /// The minor version is set as the quarter when this release happened
+    pub minor: &'static str,
```

Comment:
Thanks for documenting this aspect of the versioning!

Two things here: This value is the month that the release was made in, not the quarter, and the combination of the year and month are the major version, so the second version isn't a minor version (this is important because we can make breaking changes in releases where the month has changed but the year has not, like for our next release).